django_project/base/models/location_site.py

```diff
@@ -0,0 +1,26 @@
+# coding=utf-8
+"""Site model definition.
+
+"""
+
+from django.contrib.gis.db import models
+from base.models.location_type import LocationType
+
+
+class LocationSite(models.Model):
+    """Location Site model."""
+
+    location_type = models.ForeignKey(
+        LocationType,
+        models.SET_NULL,
+        null=True,
+    )
+    geometry = models.PointField()
+
+    # noinspection PyClassicStyleClass
+    class Meta:
+        """Meta class for project."""
+        app_label = 'base'
+
+    def __unicode__(self):
+        return u'%s' % self.location_type.name
```

Comment:
why would the site have the name of it's type (class) and not its own name? Sites have historically had a name attached to them.

Reply:
I was following the diagram here : 
![36594631-9a797702-18a7-11e8-9542-88277c3cbdcd](https://user-images.githubusercontent.com/1979569/36765085-1f76b182-1c62-11e8-8fa1-aed34b22d4ff.png)